app/src/main/res/layout-sw600dp-land/ongoing_story_card.xml

```diff
@@ -93,3 +88,4 @@
     </androidx.constraintlayout.widget.ConstraintLayout>
   </com.google.android.material.card.MaterialCardView>
 </layout>
+
```

Comment:
Revert this.

Reply:
@aggarwalpulkit596 I have reverted the file to only contain the one extra line at the end as it was in the last versioned commit by Ben Henning. I hope that is what you needed to be reverted.